src/go/cmd/setup-robot/main.go

```diff
@@ -54,6 +54,7 @@ var (
 	crSyncer       = flag.Bool("cr-syncer", true, "Set up the cr-syncer, and create a Robot CR in the cloud cluster.")
 	fluentd        = flag.Bool("fluentd", true, "Set up fluentd to upload logs to Stackdriver.")
 	fluentbit      = flag.Bool("fluentbit", false, "Set up fluentbit to upload logs to Stackdriver.")
+	log_prefix_subdomain = flag.String("log-prefix-subdomain", "", "Subdomain to prepend to Fluentbit log tag prefix.")
```

Comment:
Please go to repo root and run
```
ln -sf ../../scripts/pre-commit .git/hooks/
```
and then `git commit --amend --no-edit` to trigger the code-formatters

Reply:
https://github.com/googlecloudrobotics/core/pull/143/commits/5da6e89bea98bdd3364c6849f7f5254b965129c5